Assignment2-Python/stringCompression.py

```diff
@@ -0,0 +1,32 @@
+def compress_string(string):
+    if not string:
+        return ''
+
+    compress = []
```

Comment:
Is there a reason this is a list instead of a string? You can append to a string using `+`. The only reason to use a list here would be if you were using [list comprehension](https://www.w3schools.com/python/python_lists_comprehension.asp) to do this in one line.